certinfo.go

```diff
@@ -492,12 +492,63 @@ func CertificateText(cert *x509.Certificate) (string, error) {
 						buf.WriteString("\n")
 					}
 					if len(cert.PermittedDNSDomains) > 0 {
-						buf.WriteString(fmt.Sprintf("%16sPermitted:\n%18s%s", "", "", cert.PermittedDNSDomains[0]))
+						buf.WriteString(fmt.Sprintf("%16sPermitted:\n%18sDNS: %s", "", "", cert.PermittedDNSDomains[0]))
```

Comment:
Seems like `Permitted` will only be printed if dnsdomains is not empty. Doesn't take into account that dnsDomains might be empty while other stuff is not. Same is true for `Excluded` below.